components/workspacekit/pkg/seccomp/notify.go

```diff
@@ -106,14 +107,30 @@ func LoadFilter() (libseccomp.ScmpFd, error) {
 
 // Handle actually listens on the seccomp notif FD and handles incoming requests.
 // This function returns when the notif FD is closed.
-func Handle(fd libseccomp.ScmpFd, handler SyscallHandler) (stop chan<- struct{}, errchan <-chan error) {
+func Handle(fd libseccomp.ScmpFd, handler SyscallHandler, wsid string) (stop chan<- struct{}, errchan <-chan error) {
+	log := log.WithField("workspaceId", wsid)
+
 	ec := make(chan error)
 	stp := make(chan struct{})
 
 	handledSyscalls := mapHandler(handler)
 	go func() {
 		for {
 			req, err := libseccomp.NotifReceive(fd)
+			if err != nil {
+				if err == syscall.ENOENT {
+					log.WithError(err).Warn("failed to get notification beucase it has already been not valid anymore(the kernel sets that)")
```

Comment:
What's the purpose of the warning?

Reply:
The cause may be a query due to this error. This is to track them down.
Incidentally, libseccomp-golang also handles the error separately.
https://github.com/seccomp/libseccomp-golang/blob/c0b6cd81f16769ee69fb39d55016b98f7b232021/seccomp_internal.go#L812-L814